docs/configuration.rst

```diff
@@ -15,16 +15,67 @@ below:
      - Type
      - Default value
      - Description
-   * - ``DD_SERVICE_NAME`` or ``DATADOG_SERVICE_NAME``
+   * - ``DD_ENV``
+     - String
+     -
+     - Set an application's environment e.g. ``prod``, ``pre-prod``, ``stage``.
+   * - ``DATADOG_ENV``
+     - String
+     -
+     - Deprecated: use ``DD_ENV``
+   * - ``DD_SERVICE``
      - String
      - (autodetected)
-     - The service name to use.
+     - Set the service name to be used for this application. A default is
+       provided for these integrations: :ref:`bottle`, :ref:`flask`, :ref:`grpc`,
+       :ref:`pyramid`, :ref:`pylons`, :ref:`tornado`, :ref:`celery`, :ref:`django` and
+       :ref:`falcon`.
+   * - ``DD_SERVICE`` or ``DATADOG_SERVICE_NAME``
```

Comment:
```suggestion
   * - ``DD_SERVICE_NAME`` or ``DATADOG_SERVICE_NAME``
```
?

Reply:
if you `git grep` both are used so I am utterly confused.

I hope it's `DD_SERVICE` otherwise we can have `DD_ENV_NAME`…

Reply:
Our environment variables are such a mess 😕 😞